CHANGELOG.md

```diff
@@ -2,7 +2,8 @@
 
 ## Current (in progress)
 
-- Nothing yet
+- [BREAKING] Migrate to python >= 3.9
```

Comment:
Seems to work with my 3.8.10 version. We should update the Readme that mentions Python 3.7+ in any case.

Reply:
It _might_ work but the dependencies are explicitly computed for 3.9. Agreed for the README!